ts/store/reducers/entities/services/index.ts

```diff
@@ -116,8 +116,9 @@ export const isLoadingServicesSelector = createSelector(
 //
 
 // Check if the passed service is local or national through data included into the service metadata.
-// If service metadata aren't loaded, the service is treated as local, otherwise it returns
-// true  if service scope is equal to the filter localization parameter
+// If the localization parameter is expressed, the corresponding item is not included into section if:
+// -  the localization paramenter is different to the service scope
+// -  service metadata load fails,
 const hasLocalization = (
```

Comment:
```suggestion
const isInScope = (
```
Because this PR is regarding the scope checking, what do you think to name it scope instead of localization?

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1361/commits/f2df9e948a68544775b08349cb240c4671afd085